lib/src/protocol_instance.rs

```diff
@@ -188,11 +190,15 @@ fn get_blob_proof_type(
     proof_type: VerifierType,
     blob_proof_type_hint: BlobProofType,
 ) -> BlobProofType {
-    match proof_type {
-        VerifierType::None => blob_proof_type_hint,
-        VerifierType::SGX => BlobProofType::ProofOfCommitment,
-        VerifierType::SP1 => BlobProofType::ProofOfEquivalence,
-        VerifierType::RISC0 => BlobProofType::ProofOfEquivalence,
+    if cfg!(feature = "proof_of_equivalence") {
```

Comment:
Although it's merged but making a note:
should not flag at this location, this function only controls what's run within PI hash, it does not impact the output anyway. If we want to temporariliy disable POE in PI then just flag PI.